napari_aicssegmentation/view/workflow_select_view.py

```diff
@@ -1,63 +1,217 @@
+        pass
+
+    def _load_workflows(self, workflows):  # workflows: List[aicssegmentation.WorkflowStep]
+        # TODO generate workflow grid from list of workflows
+        # -> https://github.com/AllenCell/napari-aicssegmentation/issues/26
+        pass
+
+    def _reset_combo_box(self, combo: QComboBox):
+        """
+        Reset a combo box to its original state, keeping the header but removing all other items
+        """
+        if combo.count() > 0:
+            header = combo.itemText(0)
+            combo.clear()
+            combo.addItem(header)
+
+    def _dropdown_row(self, number: int, placeholder: str, enabled=False) -> FormRow:
+        """
+        Given the contents of a dropdown and a number for the label, return a label and a QComboBox
+        widget that can be used to create a row in a QFormLayout
+        """
+        label = f"{number}."
+
+        dropdown = QComboBox()
+        dropdown.addItem(placeholder)
+        dropdown.setDisabled(not enabled)
+
+        return FormRow(label, dropdown)
+
+    # TODO turn into an actual reusable widget if this is needed anywhere else
+    def _form_layout(self, rows: List[FormRow], margins=(0, 5, 11, 0)) -> QFrame:
+        """
+        Create a nicely formatted form layout given contents to add as rows.
+
+        Inputs:
+            rows:       List of FormRow
+            margins:    Tuple of 4 numbers representing left, top, right, and bottom margins for
+                        the form's contents. Qt defaults to (11, 11, 11, 11).
+        Output:
+            A QFrame widget with a QFormLayout
+        """
+        widget = QFrame()
+        layout = QFormLayout()
+        layout.setFormAlignment(QtCore.Qt.AlignLeft)
+        left, top, right, bottom = margins
+        layout.setContentsMargins(left, top, right, bottom)
+
+        for row in rows:
+            layout.addRow(row.label, row.widget)
+        widget.setLayout(layout)
+        return widget
+
+    def _add_step_3_layout(self, layout: QLayout, enabled=False):
+        """
+        Add widgets and set the layout for the Step 3 instructions and the workflow buttons
+        """
+        step_3_label = QLabel("3.")
+        step_3_label.setAlignment(QtCore.Qt.AlignTop)
+        step_3_instructions = QLabel(
+            "Click a button below that most closely resembles your image channel to select & start a workflow"
+        )
+        step_3_instructions.setWordWrap(True)
+        step_3 = self._form_layout([FormRow(step_3_label, step_3_instructions)], (0, 0, 11, 0))
+
+        if enabled is False:
+            step_3_instructions.setObjectName("step3InstructionsDisabled")
+        layout.addWidget(step_3)
+
+        # Row of text labeling the columns of workflow images
+        column_labels = QWidget()
+        column_layout = QHBoxLayout()
+        column_layout.setContentsMargins(11, 11, 11, 0)
+        column_labels.setLayout(column_layout)
```

Comment:
Another styling non-transfer (I imagine it has to do with replacing the style_constants file with a proper QSS file which removed the dimension constants): Need to add `column_labels.setFixedWidth(400)` here. Otherwise the column labels aren't positioned correctly.

I think we should give these constants a home and use them instead if possible, though.

```
""" Dimensions """
PAGE_WIDTH = 440
PAGE_CONTENT_WIDTH = PAGE_WIDTH - 40
WORKFLOW_BUTTON_HEIGHT = 200
```